doc/radosgw/notifications.rst

```diff
@@ -50,25 +50,25 @@ In this case, the only latency added to the original operation is of committing
 Topic Management via CLI
 ------------------------
 
-Configuration of all topics of a user could be fetched using the following command:
+Configuration of all topics of (per tenant) could be fetched using the following command:
```

Comment:
accessible by a tenant?

Reply:
this is actually a change from here: https://github.com/ceph/ceph/pull/38136
will fix there.